shared/locales/firefox_sync/firefox_sync.en-US.properties

```diff
@@ -2,6 +2,7 @@ fxsync=Firefox Sync
 fxsync-collections-bookmarks=Bookmarks
 fxsync-collections-history=History
 fxsync-collections-passwords=Passwords
+fxsync-empty-account=This account is empty. You need to sync from Firefox, Firefox for Android or Firefox for iOS first and try again here.
```

Comment:
Instead of "... try again here", maybe it is more clear: " ... and then sync again from here".